install.sh

```diff
@@ -87,11 +87,19 @@ quote() {
   done
 }
 
-# Add indentation and trailing slash to quoted args
+# Add indentation and trailing slash to quoted args except last one
+# Also don't add trailing slash to command if no args are given
 quote_indent() {
-  printf ' \\\n'
+  if [ $# -ge 1 ]; then
+    printf ' \\\n'
+  fi
+  _arg_count=1
```

Comment:
You have initialized `_arg_count=1` but as far as I can see it's not updated anywhere. Am I missing something?

Reply:
Well... yes ;) Tested only with one argument. Fixed